src/wp-includes/class-wp-theme-json.php

```diff
@@ -1606,7 +1607,6 @@ public function get_svg_filters( $origins ) {
 
 			$duotone_presets = $node['color']['duotone'];
 
-			$filters = '';
```

Comment:
as per my comment in the [trac's ticket](https://core.trac.wordpress.org/ticket/55241#comment:12), I'll suggest to not remove this line... this will keep the original behavior(which mean the variable will be initializing each time the loop is running). thanks.